### DISCUSSION TITLE
文档和rem有什么区别？

### DISCUSSION THREAD

**Discussion by Lorien95** [5eagull/RemNote-Chinese-chat-group#42] [你问我答/互相帮助]
如何判断一个rem要不要转为文档呢

**Comment by makerjackie** [comment]
如果里头的内容很多，或者你希望搜索的时候优先展示，可以转换为文档（快捷键是 ctrl shift alt + D）

**Reply by MyEnthusiastic** [reply]
有道理，我一直都不怎么使用文档 因为我太喜欢everything is a rem这个概念了

**Reply by MyEnthusiastic** [reply]
有道理，我一直都不怎么使用文档 因为我太喜欢everything is a rem这个概念了
